### DISCUSSION TITLE
General WARNING

### DISCUSSION THREAD

**Discussion by Samthibau** [vercel/community#4105] [Help] [answered]
### Summary

When creating a team watch out, 

For some reason, I guess they must have missed this in development, but it's not clearly stated anywhere on the Create Git Repository page that they are going to charge you $20.

This is misleading as other platforms just want the information to verify that you are real, and since it is not stated that you are being charged by Vercel most users are going to be misled by this extremely dumb and very illegal mistake.

As for the refund, I'm going on 8 hours now with 0 response from any support, i even tried contacting the "Super fast abuse line" that was 5 hours ago and i have also received 0 response.

honestly might be best to avoid Vercel altogether until they do some restructuring. 

Note this is after the free trial expires, they do technically state that they charge $20 but it is kind of hidden and most people discover it after the fact.

the only place its really stated is here https://vercel.com/docs/accounts/plans/pro  in small slightly bold words otherwise its basically mentioned nowhere else on the site.

### Steps to Reproduce

go to https://vercel.com/new  then click on any of the clone templates, if you scroll down a bit you will see the mislabeled Create a team box.Team

**Comment by swarnava** [comment, marked as answer]
Hi Sam,

Our Pro plan charges are mentioned in the documentation as well as pricing page here: https://vercel.com/pricing. When you create a team, we do mention that we give 14 days trial for free and they are chargable:

<img width="1256" alt="image" src="https://github.com/vercel/community/assets/1162991/e5920b73-a401-46a0-bac8-126a9c75c26e">



As of the refund request, we currently experiencing high volume of request and they will be prioritized accordingly. We will triage your request and get back to you soon.
